seqr/views/apis/summary_data_api.py

```diff
@@ -182,19 +182,31 @@ def bulk_update_family_external_analysis(request):
 
 def _load_aip_data(data: dict, user: User):
     category_map = data['metadata']['categories']
+    projects = data['metadata'].get('projects')
     results = data['results']
 
-    family_id_map = dict(Individual.objects.filter(
-        family__project__in=get_internal_projects(), individual_id__in=results.keys(),
-    ).values_list('individual_id', 'family_id'))
+    if not projects:
+        raise ErrorsWarningsException(['No projects specified in the metadata'])
+
+    family_id_map = defaultdict(list)
+    for individual_id, family_id in Individual.objects.filter(
+        family__project__in=get_internal_projects().filter(name__in=projects), individual_id__in=results.keys(),
+    ).values_list('individual_id', 'family_id'):
+        family_id_map[individual_id].append(family_id)
+    errors = []
     missing_individuals = set(results.keys()) - set(family_id_map.keys())
     if missing_individuals:
-        raise ErrorsWarningsException([f'Unable to find the following individuals: {", ".join(sorted(missing_individuals))}'])
+        errors.append(f'Unable to find the following individuals: {", ".join(sorted(missing_individuals))}')
+    multi_family_individuals = {individual_id for individual_id, families in family_id_map.items() if len(families) > 1}
+    if multi_family_individuals:
+        errors.append(f'The following individuals are found in multiple families: {", ".join(sorted(multi_family_individuals))}')
```

Comment:
Why can't we load aip data if an individual is in multiple families?

Reply:
I see in the upload fixture that the results are just keyed by individual with no family ID. Maybe the AIP results format should change to include family ids so that we can capture all of the AIP results.

Reply:
AIP results are only generated for one specific family, so if multiple families match that is ambiguous and we are currently randomly assigning the results to one, instead of having the user provide disambiguation. Family IDs are not unique across projects so having the AIP upload include that data does not really fix the issue